app/lib/hooks/useLocaleFromParams.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { localeExists } from "lib/i18n";
+
+/**
+ * For the HOME view in the app. On mount, validates, extracts and strips the locale params from the url and returns the value from the locale param.
+ * @example https://app.klimadao.finance/#/stake?locale=fr => "fr" or https://app.klimadao.finance/#/stake?locale=not-supported-language => null
```

Comment:
Based on this comment, I don't think the website is linking to the dapp correctly.

For the 2nd test, the staging website links to the prod app, not the staging app, so not sure if the link created is correct.

It links to the following for German: https://app.klimadao.finance/?locale=de#/stake
Shouldn't it be the following? https://app.klimadao.finance/#/stake?locale=de

Reply:
Thanks @ShimonD-KlimaDAO 

This got fixed in this [PR here](https://github.com/KlimaDAO/klimadao/pull/339)